Classes/Common/Solr.php

```diff
@@ -110,7 +110,7 @@ public static function escapeQuery($query) {
         $helper = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\Solarium\Core\Query\Helper::class);
         // Escape query phrase or term.
         if (preg_match('/^".*"$/', $query)) {
-            return '"'.$helper->escapePhrase(trim($query, '"')).'"';
+            return $helper->escapePhrase(trim($query, '"'));
```

Comment:
Why did you remove the quotes? Without quotes phrases would become simple multi-term searches.

Reply:
This affects every search, not only 'Search in Document'.

Reply:
This should have been another pull-request as this is a bug in Kitodo.Presentation since the switch to Solarium. At the moment, Kitodo.Presentation can't do phrase searches :-(

Example:

current patch
-----------------
The $query arrives at line 113 with this content:
`$query=""festung dresden""`
With trim($query, '"') we get
`$query = "festung dresden"`
With Solarium escapePhrase adds the Quotes again and we return at this point:
`return ""festung dresden""`

This results finally in the folowing query in case of the searchInDocument eID-script:
`http://solrserver:8983/solr/dlfCore2/select?wt=json&q=fulltext:(%22festung%20dresden%22)%20AND%20uid:11&hl=on&hl.fl=fulltext&fl=uid,id,page&hl.method=fastVector&start=0&rows=20`

Solr sees the following query parameter and executes a phrase search:
`"q":"fulltext:(\"festung dresden\") AND uid:11",`

Result textsnippet e.g.:
`"fulltext":["17. April 1945 Seite 19 Hermann Rahne Die »<em>Festung Dresden</em>« Seite 32 Christian Hermann Millionen Krie"]},`

current master
-------------------
With the additional quotes, the result would be:
`http://solrserver:8983/solr/dlfCore2/select?wt=json&q=fulltext:(%22%22festung%20dresden%22%22)%20AND%20uid:11&hl=on&hl.fl=fulltext&fl=uid,id,page&hl.method=fastVector&start=0&rows=20`

Solr sees the following query parameter and does a simple term search:
`"q":"fulltext:(\"\"festung dresden\"\") AND uid:11",`

Resulting textsnippet e.g.:
` "fulltext":["19 Hermann Rahne Die »<em>Festung</em> <em>Dresden</em>« von 1945 Am 16. April 1945 konnten die Dresdner Bürger in der"]},`

Reply:
So the problem seems to be: Why is the query double-quoted in the first place?

Reply:
It's not double quoted in the first place. That's only the output of xdebug.

The question is, why add the quotes another time? This is done already by escapePhrase(). This is what is does:
<pre>
public function escapePhrase($input)
{
return '"' . preg_replace('/("|\\\)/', '\\\$1', $input) . '"';
}
</pre>

Reply:
OK, got it!